src/pages/console/authz/permissions.mdx

```diff
@@ -16,26 +16,26 @@ You are going to define set up role-based authorization rules for each of the mo
 2. In the **Model** pane on the right, expand the **Owners** window. 
 3. Choose **Create**, **Read**, **Update** and **Delete** to specify that *Owners* have create, read, update, and delete access. The settings look as follows.
 
-![GSA](/images/console/10_ownersaccess.png)
+![Owner authorization rules on a GraphQL data model](/images/console/10_ownersaccess.png)
 
 ## To set a private authorization rule
 1. Using the *Books* data model that you created in the [Create a data model example](/console/data/data-model#Create-a-data-model-example), set the authorization mode to **Cognito user pool**. 
 2. In the **Model** pane on the right, expand the **Any signed-in users** window. 
 3. Choose **Create**, **Read**, and **Update** to specify that any signed-in authenticated user has create, read, and update, access. The settings look as follows.
 
-![GSA](/images/console/11_privatesaccess.png)
+![Private authorization rules on a GraphQL data model](/images/console/11_privatesaccess.png)
 
 
 ## To set a group authorization rule
 1. Using the *Books* data model that you created in the [Create a data model example](/console/data/data-model#Create-a-data-model-example), set the authorization mode to **Cognito user pool**. 
 2. Create an *Editors* group using the instructions to [create a group](/console/auth/user-management#To-create-a-group). Alternately, you can create a new group from the **Add a new rule for...** menu.
 3. In the **Model** pane on the right, select *Editors* from the **Add a new rule for...** menu. 
 
-![GSA](/images/console/8_menudetaileditors.png)
+![Process of adding group authorization rules by selecting "Add a new rule" in the menu on a GraphQL data model](/images/console/8_menudetaileditors.png)
```

Comment:
This could probably be decorative as well since the step 3 above describes it well. if you want to keep it then that's okay too. You could do something like this:

```suggestion
![Screenshot of step 3 as mentioned abovel](/images/console/8_menudetaileditors.png)
```

I got that idea from this resource: https://it.ucsf.edu/how-to/accessible-images-best-practices, there's a section called `Screen shots for "How To" documentation`